### PR TITLE
Milestone 2: Create - Remove book

### PR DESCRIPTION
**Pull request by Kingjosh007**
In this milestone, I followed [these instructions](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md) in order to implement the Add and Delete features.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
-    <li className="single-book">
+    <li className="single-book" id={elementId}>
       <div className="left-side">
         <div className="book-details">
-          <div className="book-category">{book.category}</div>
-          <div className="book-title">{book.title}</div>
-          <div className="book-category">{book.author}</div>
+          <div className="book-category">{category}</div>
+          <div className="book-title">{title}</div>
+          <div className="book-category">{author}</div>
```
Comment (mikenath223) [inline src/components/Book.js:15]: - [ ] On clicking the add book button nothing gets added. Please fix this.

File: src/components/Book.js [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
         <div className="book-buttons">
           <button type="button" className="comments-btn">Comments</button>
           <div className="separator">|</div>
-          <button type="button" className="remove-btn">Remove</button>
+          <button
+            type="button"
+            className="remove-btn"
+            onClick={handleDelete}
+          >
+            Remove
```
Comment (mikenath223) [inline src/components/Book.js:25]: - [ ] Also ensure the remove book functionality works.

File: src/components/Book.js (hunk 1 above)
Comment (Kingjosh007) [reply src/components/Book.js:15]: Fixed. Thanks for pointing it out.

File: src/components/Book.js (hunk 2 above)
Comment (Kingjosh007) [reply src/components/Book.js:25]: Fixed. Thanks for pointing it out.

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @Kingjosh007 

Amazing job implementing suggestions from the previous reviewer! 💯 
Congratulations! 🎉 



## Aspects to improve ♻️
_Check the comments under the review._
- [OPTIONAL] It would be better to fix the broken MIT link in your README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**